app/controllers/authorities_controller.rb

```diff
@@ -8,7 +8,11 @@ def query
       if params[:term] == "location"
         GeoNamesResource.find_location(s)
       else
-        LocalAuthority.entries_by_term(params[:model], params[:term], s) rescue []
+        begin
```

Comment:
[B] this is the code that I referenced in [A] 

I think the multi-line code should be always preferred so that people don't have to guess (is a one-liner OK in `if` statements but not `rescue` operations or viceversa?) but as long as a tool applies the criteria evenly we should be OK.